barretenberg/cpp/src/barretenberg/circuit_checker/standard_circuit_builder.test.cpp

```diff
@@ -1,4 +1,5 @@
-#include "standard_circuit_builder.hpp"
+#include "barretenberg/proof_system/circuit_builder/standard_circuit_builder.hpp"
```

Comment:
Feels weird that the tests for builders are here and not in the same folder as the implementation